src/servicenow/servicenow-api.ts

```diff
@@ -28,7 +29,7 @@ export class ServiceNowApi {
 
     const json = (await response.json()) as ServiceNowResponse;
     const end = json.result.meta.end;
-    let list = json.result.articles as ServiceNowArticle[];
+    let list = json.result.articles;
 
     if (json.result.meta.count > end) {
       const nextUrl = `/api/sn_km_api/knowledge/articles?fields=category,text,workflow_state,topic&limit=${this.limit}&offset=${end}`;
```

Comment:
if you add category and topic, then it would be good to add kb_category as well to the fields.
category and topic are knowledge management v2
kb_category is v3

Reply:
fixed